components/FriendPages.js

```diff
@@ -28,6 +28,9 @@ const StyledName = styled.p`
 `;
 
 export default function FriendPages() {
+  const [data, setData] = useLocalStorageState("friendsApp", {
+    defaultValue: defaultData,
+  });
```

Comment:
You can work here with the props data if you follow my remark above
```suggestion
export default function FriendPages({data}) {

```